.drone.yml

```diff
@@ -4345,7 +4345,7 @@ trigger:
     - gravitational/teleport
   branch:
     include:
-    - master
+    - branch/v7
```

Comment:
This change isn't related to Go upgrade, is it?

Reply:
The push trigger needs to be on `branch/v7` rather than master, otherwise the `1.16.14` image will never be built (this pr targets `branch/v7`).